src/components/TextInput/TextInput.tsx

```diff
@@ -219,6 +229,32 @@ export const TextInput: FC<TextInputProps> = ({
                     <Spinner />
                 </span>
             )}
+            {copyable && (
+                <button
+                    className={merge([
+                        "tw-flex tw-items-center tw-justify-center tw-transition-colors tw-rounded",
+                        disabled ? "tw-cursor-default tw-text-black-40" : "tw-text-black-60 hover:tw-text-black-100",
+                        copyButtonIsFocusVisible && FOCUS_STYLE,
+                    ])}
+                    onClick={() => copy(value)}
+                    data-test-id="copy-icon"
+                    title="Copy input value"
```

Comment:
Just for confirmation: Is this the final title?

Reply:
I changed it to "copy input text", but if you have any other ideas I would appreciate it :)